src/Tasks/Microsoft.Common.CrossTargeting.targets

```diff
@@ -136,7 +136,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     This will import NuGet restore targets. We need restore to work before any package assets are available.
   -->
   <PropertyGroup>
-    <NuGetRestoreTargets Condition="'$(NuGetRestoreTargets)'=='' and '$([MSBuild]::IsRunningFromVisualStudio())'=='true'">$(MSBuildToolsPath32)\..\..\..\Common7\IDE\CommonExtensions\Microsoft\NuGet\NuGet.targets</NuGetRestoreTargets>
+    <MSBuildUseVisualStudioDirectoryLayout Condition="'$(MSBuildUseVisualStudioDirectoryLayout)'==''">$([MSBuild]::IsRunningFromVisualStudio())</MSBuildUseVisualStudioDirectoryLayout>
```

Comment:
I wanted to say that this would be guaranteed to be already set by common.props, but I couldn't prove it to myself so I'm ok leaving the redundancy here. It shouldn't be expensive since it gets conditioned out if already set anyway.